client/packages/coldchain/src/Monitoring/ListView/TemperatureBreach/TemperatureBreachList.tsx

```diff
@@ -18,6 +21,41 @@ import {
 import { BreachTypeCell } from '../../../common';
 import { Toolbar } from './Toolbar';
 
+const DurationCell = ({ rowData }: CellProps<TemperatureBreachFragment>) => {
+  const t = useTranslation('coldchain');
+  const { localisedDistance } = useFormatDateTime();
+  const duration = !rowData.endDatetime
+    ? t('label.ongoing')
+    : localisedDistance(rowData.startDatetime, rowData.endDatetime);
```

Comment:
using the start and end dates, so that I can use the `formatDistance` fn from date-fns. That's easier than converting the milliseconds to something that `formatDuration` will accept.
It also means that what the user can see is the start, end, duration : all of which will be consistent with each other.

Reply:
Sounds good.

> It also means that what the user can see is the start, end, duration : all of which will be consistent with each other.

Although I think this will be consistent right ? (mSupply Cold Chain would send it correctly ?) And I guess filtering would work on duration column in database and I guess it should still be consisten ?

Reply:
it will be consistent yeah. and there is no sort or filter on duration currently.. so we are ok with that